### ISSUE TITLE
SCRIPT1

### ISSUE THREAD

**Issue by Abidinen** [Tampermonkey/tampermonkey#2282]
(Please fill <!DOCTY PE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>حجز موعد فيزا</title>
    <style>
        body {
            font-family: Arial, sans-serif;
            background-color: #f4f4f9;
            margin: 0;
            padding: 0;
        }

        .container {
            max-width: 600px;
            margin: 50px auto;
            background: #fff;
            padding: 20px;
            border-radius: 8px;
            box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
        }

        h1 {
            text-align: center;
            color: #333;
        }

        form {
            display: flex;
            flex-direction: column;
        }

        label {
            margin-bottom: 5px;
            font-weight: bold;
            color: #555;
            font-size: 16px;
        }

        input[type="text"],
        input[type="password"],
        button {
            padding: 10px;
            margin-bottom: 15px;
            border: 1px solid #ccc;
            border-radius: 5px;out the issue template with your details)

## Specifications

- Chromium: (i.e. 64.0.3282.140)
- TM: (i.e. 4.5.5648)
- OS: (i.e. MacOS 10.13.4)

## Script

(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name MyName
// @namespace http://mywebsite.com
// @Version 1.0
// @description try to take over the world!
// @author Me
// @match http://mywebsite.com/*
// @require http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js
// @include https://mywebsite.com/*
// @grant none
// ==/UserScript==
```
      font-size: 16px;
        }

        .instruction {
            font-size: 14px;
            color: #555;
            margin-bottom: 10px;
        }

        button {
            background-color: #007bff;
            color: white;
            border: none;
            cursor: pointer;
        }

        button:hover {
            background-color: #0056b3;
        }

        .error {
            color: red;
            font-size: 14px;
        }
    </style>
</head>
<body>
    <div class="container">
        <h1>تسجيل الدخول لحجز الموعد</h1>
        <div class="instruction"> abidinen@gmail.com املأ اسم المستخدم في الحقل المخصص. </div>
        <div class="instruction"> Abdine1988@ أدخل كلمة المرور في الحقل المخصص. </div>
        <form id="loginForm">
            <label for="username">abidinenine@gmail.com  اسم المستخدم</label>
            <input type="text" id="username" placeholder="أدخل اسم المستخدم" required>

            <label for="password"> Abidine1988@ كلمة المرور</label>
            <input type="password" id="password" placeholder="أدخل كلمة المرور" required>

            <div id="errorMessage" class="error"></div>

            <button type="submit">  تسجيل الدخول</button>
        </form>
    </div>

    <script>
        document.getElementById('loginForm').addEventListener('submit', function(e) {
            e.preventDefault();

            const username = document.getElementById('username').value;
            const password = document.getElementById('password').value;
            const errorMessage = document.getElementById('errorMessage');

            errorMessage.textContent = '';

            if (!username || !password) {
                errorMessage.textContent = 'يرجى ملء جميع الحقول.';
                return;
            }

            fetch('https://algeria.blsspainglobal.com/DZA/account/login', {
                method: 'POST',
                headers: {
                    'Content-Type': 'application/json',
                },
                body: JSON.stringify({ username, password }),
            })
                .then(response => {
                    if (response.ok) {
                        alert('تم تسجيل الدخول بنجاح.');
                        window.location.href = 'https://algeria.blsspainglobal.com/DZA/Appointment/VisaType';
                    } else {
                        errorMessage.textContent = 'فشل تسجيل الدخول. تحقق من بياناتك.';
                    }
                })
                .catch(error => {
                    errorMessage.textContent = 'حدث خطأ أثناء الاتصال بالخادم. حاول مرة أخرى لاحقًا.';
                });
        });
    </script>
</body>
</html>
